service/ui_plugin_build_baron.go

```diff
@@ -114,8 +114,8 @@ func (uis *UIServer) bbJiraSearch(rw http.ResponseWriter, r *http.Request) {
 	taskId := vars["task_id"]
 	exec := vars["execution"]
 
-	searchReturnInfo, projectNotFound, err := graphql.GetSearchReturnInfo(taskId, exec)
-	if projectNotFound {
+	searchReturnInfo, bbConfig, err := graphql.GetSearchReturnInfo(taskId, exec)
+	if !bbConfig.ProjectFound {
```

Comment:
do we need to address `!SearchConfigured` here as well?

Reply:
I don't think so, but I realized I need to reconfigure it [here](https://github.com/evergreen-ci/evergreen/blob/master/public/static/plugins/buildbaron/partials/task_build_baron.html#L3), so thanks for pointing that out.